### PR TITLE
Clean list of Operations supported by fence and quiet

### PR DESCRIPTION
**Pull request by naveen-rn**
Fix for https://github.com/openshmem-org/specification/issues/278

### CODE REVIEW

**Comment by naveen-rn** [comment]
@nspark @BryantLam - Can you verify whether this change makes sense? This is the initial take on the
enumeration issue - we can update it as required.

File: content/shmem_quiet.tex [hunk 1]
```diff
@@ -38,23 +34,21 @@
...
     symmetric data objects on all \acp{PE}.
 
     In an \openshmem program with multithreaded \acp{PE}, it is the
-    user's responsibility to ensure ordering between operations issued by the threads
-    in a \ac{PE} that target symmetric memory (e.g. \PUT{}, \ac{AMO}, \OPR{put-with-signal}, memory stores,
-    and nonblocking routines) and calls by threads in that \ac{PE} to
+    user's responsibility to ensure ordering between operations issued by the
+    threads in a \ac{PE} that target symmetric memory
+    (e.g. \PUT{}, \ac{AMO}, \OPR{put-with-signal}, memory stores,
+    and nonblocking routines)
```
Comment (BryantLam) [inline content/shmem_quiet.tex]: What's the reason for why this list of operations in parentheses were left in when the others were removed?

File: main_spec.tex [hunk 2]
```diff
@@ -475,6 +475,30 @@ \subsection{Memory Ordering Routines}\label{subsec:memory_order}
 \OPR{put-with-signal}, \GET{}, and \ac{AMO} routines to symmetric data
 objects.
```
Comment (BryantLam) [inline main_spec.tex]: This description is now redundant with the table. Can you shorten it similar to how you shorted the phrasing in the other changes?

Comment (BryantLam) [reply main_spec.tex]: Also, there might be other places like this. It looks like you caught most of them, but you could further check by e.g., grep'ing for things like "memory store".

**Comment by jdinan** [comment]
Please move this PR to the relevant section committee's branch.

**Pull request closed without merging**